src/styles/fonts.css

```diff
@@ -38,11 +40,3 @@
 	size-adjust: 112%;
 	line-gap-override: 19%;
 }
-
-@font-face {
-	font-family: obviously-wide-fallback;
-	src: local("Arial Black");
-	size-adjust: 134%;
-	ascent-override: 96%;
-	descent-override: 27%;
-}
```

Comment:
Worth double checking this — we put it in place to reduce CLS when the heavier weight Obviously loaded in vs system fonts.